src/cssjanus.js

```diff
@@ -119,7 +119,7 @@ function CSSJanus() {
 		colorPattern = '(#?' + nmcharPattern + '+|(?:rgba?|hsla?)\\([ \\d.,%-]+\\))',
 		urlCharsPattern = '(?:' + urlSpecialCharsPattern + '|' + nonAsciiPattern + '|' + escapePattern + ')*',
 		lookAheadNotLetterPattern = '(?![a-zA-Z])',
-		lookAheadNotOpenBracePattern = '(?!(' + nmcharPattern + '|\\r?\\n|\\s|#|\\:|\\.|\\,|\\+|>|\\(|\\)|\\[|\\]|=|\\*=|~=|\\^=|\'[^\']*\'])*?{)',
```

Comment:
Can you explain the removal of the closing square bracket `]` from the original line? Actually, I'm unsure as to why it what was there, but would be good to confirm whether its removal was important to this change, or just clean up, or something else.

Reply:
I don't actually remember the reason for that, but I can guess.

Presumably, the original purpose of the `]` was to ensure that the quotes were specifically part of simple attribute selector values, like `[class='span']` (`']` at the end), which are by far the most common use of quotes in selectors. However, this prevents it from matching other types of quotes in selectors, such as `:lang("en")` and `[class='span' i]`. (Currently, both have some browser support and are in Working Draft state of the spec.)